### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by benwmx**
## Details 
In this pull request I added the following :
* Create owners table.
* Create species table.
* Modify animals table to include species_id & owner_id column.
* Insert owner's data.
* Insert species data.
* Update animals table to include species_id & owner_id data.
* Write Queries to answer analytical questions.

## ScreenShot
pr3

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
-    species VARCHAR(255),
+    name VARCHAR(50),
     date_of_birth DATE,
     escape_attempts INT,
     neutred BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species_id INT REFERENCES species(id),
+    owner_id INT REFERENCES owners(id)
 );
```
Comment (Lameck1) [inline schema.sql:23]: - [ ] You have done modifying this table as per requirement. However, this task is testing our ability to alter existing database tables not rewriting them. In this task, you are supposed to use the `ALTER TABLE` SQL command to alter this table.

Example of ALTER TABLE usage:

```sql
ALTER TABLE ADD name_of_column_to_add;
```

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @benwmx, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benwmx** [comment]
Thank you 🙏🏽  @Lameck1  for your thorough feedback I appreciate that.
I implemented all the changes that you requested.

**Comment by Lameck1** [comment]
You are very much welcome @benwmx.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @benwmx, 

Great work with making the changes requested by the previous code reviewer.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**